<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="folder_btn">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Відкрити папку</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/user/Downloads/folder.svg</normaloff>C:/Users/user/Downloads/folder.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="image_list">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="del_btn">
          <property name="text">
           <string>Видалити</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>C:/Users/user/Downloads/trash.svg</normaloff>C:/Users/user/Downloads/trash.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="text">
           <string>Зберегти</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>C:/Users/user/Downloads/folder-up.svg</normaloff>C:/Users/user/Downloads/folder-up.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_btn">
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forward_btn">
          <property name="text">
           <string>Вперед</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="rotate_left">
          <property name="text">
           <string>Поворот Л</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rotate_right">
          <property name="text">
           <string>Поворот Р</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="current_image">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Картинка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_folder"/>
    <addaction name="open_file"/>
    <addaction name="save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редагування</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Ефекти</string>
    </property>
    <addaction name="separator"/>
    <addaction name="black_white"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Корекція</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="sharpen"/>
    <addaction name="blur"/>
    <addaction name="action_16"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Скасувати</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Скинути</string>
   </property>
  </action>
  <action name="open_folder">
   <property name="text">
    <string>Відкрити папку</string>
   </property>
  </action>
  <action name="open_file">
   <property name="text">
    <string>Відкрити файл</string>
   </property>
  </action>
  <action name="black_white">
   <property name="text">
    <string>Чорно Біле</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Контраст</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Різкість</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Яскравість</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Насиченість</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Розмиття</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Контрасність</string>
   </property>
  </action>
  <action name="sharpen">
   <property name="text">
    <string>Різкість</string>
   </property>
  </action>
  <action name="blur">
   <property name="text">
    <string>Розмиття</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Насиченість</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
